sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/global-namespace-with-jsdom-unit-testing_sjs-0.6/build.sbt

```diff
@@ -1,35 +0,0 @@
-
```

Comment:
There's no need for any equivalent of the global-namespace tests. They were 0.6.x-only because the global namespace hackery that was done there was only necessary in Scala.js 0.6.x in the first place. Scala.js 1.x has a sane handling of the global namespace.